Source/WebCore/platform/image-decoders/gif/GIFImageReader.h

```diff
@@ -194,7 +194,7 @@ struct GIFFrameContext {
         m_lzwBlocks.append(GIFLZWBlock(position, size));
     }
 
-    bool decode(const unsigned char* data, size_t length, WebCore::GIFImageDecoder* client, bool* frameDecoded);
+    bool decode(std::span<const uint8_t> data, WebCore::GIFImageDecoder* client, bool& frameDecoded);
```

Comment:
If you are making this out parameter required, you should consider changing the return type to a struct that can represent both outputs.

Reply:
It was already required, so passing a pointer seemed confusing.

Reply:
I've been converting out arguments into return values on functions like this, but it does make the refactoring more risky so I wouldn't insist on it.